pandas/tests/series/test_replace.py

```diff
@@ -249,3 +249,12 @@ def test_replace_mixed_types_with_string(self):
         result = s.replace([2, '4'], np.nan)
         expected = pd.Series([1, np.nan, 3, np.nan, 4, 5])
         tm.assert_series_equal(expected, result)
+
+    @pytest.mark.parametrize('to_replace', [pd.NaT, [np.nan, pd.NaT]])
+    def test_replace_with_tz_aware_data(self, to_replace):
```

Comment:
can you tests with tz-naive as well, maybe just have a combined test (as we likely have one for naive)